generated/nifgen/tox-system_tests.ini

```diff
@@ -47,6 +47,8 @@ deps =
     nifgen-system_tests: hightime
     nifgen-system_tests: fasteners
     nifgen-system_tests: pytest-json
+    nifgen-system_tests: grpcio
```

Comment:
This tells me that the wheel for `nifgen` is not going to install these things, so the client that wants to use `nifgen` with gRPC needs to explicitly install them.

Do we provide reasonable errors if this isn't the case? Maybe it's a moot point because you need these in order to create the channel?

Reply:
Correct, the client would need to explicitly install them. If you think we should auto-install it via dependencies, we can have that conversation.

You're right that the user has to create the channel, at least for now certainly.

Reply:
No I think this is good.
We did the same for numpy.

Reply:
Yeah the channel creation is up to the client, so they'll need the python gRPC package installed. They would likely get some nonsensical python error if they passed in `_grpc_channel=True` but that's fine with me.